src/reconcile/rec_write.c

```diff
@@ -2700,6 +2693,7 @@ __rec_hs_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r)
         }
 
 err:
+    /* If there is no work to do __wt_hs_insert_updates() is not called and this returns 0. */
```

Comment:
I don't think we need this comment. But I'm not too fussed about it, if you want you can keep it.

Reply:
I will remove it.